inspire/modules/authors/recordext/fields/author.cfg

```diff
@@ -0,0 +1,363 @@
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+@override
+name:
+    """Contains name information.
+
+    Please note that MARC field for an author's name is splitted into two
+    fields, `last_name` and `first_name`. The same situation occurs for
+    the date fields, in JSON it is splitted into `start_year` and `end_year`.
+
+    Admissible string values for `100__g`:
+    + active
+    + departed
+    + retired
+    + deceased
+
+    The only accepted value in `100__c` field is:
+    + Sir
+
+    Values accepted for `100__b:
+    + Jr.
+    + Sr.
+    + roman numbers (like VII)
+
+    Example:
+    +---------+---------------------+
+    | 100__a  | Smith, John         |
+    +---------+---------------------+
+    | 100__b  | Jr.                 |
+    +---------+---------------------+
+    | 100__c  | Sir                 |
+    +---------+---------------------+
+    | 100__d  | 1943-2001           |
+    +---------+---------------------+
+    | 100__g  | active              |
+    +---------+---------------------+
+    | 100__q  | Johnny Smith        |
+    +---------+---------------------+
+    """
+
+    schema:
+        {'name': {'type': 'dict', 'required': True,
+         'schema': {'last': {'type': 'string', 'empty': False},
+                    'first': {'type': 'string', 'empty': False},
+                    'numeration': {'type': 'string', 'nullable': True,
+                                   'regex': __import__('flask').current_app.config['INSPIRE_NAME_NUMERATIONS']},
+                    'title': {'type': 'string', 'nullable': True,
+                              'allowed': __import__('flask').current_app.config['INSPIRE_NAME_TITLES']},
+                    'birth_year': {'type': 'integer', 'nullable': True,
+                                   'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                                   'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+                    'death_year': {'type': 'integer', 'nullable': True,
+                                   'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                                   'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+                    'previous_name': {'type': 'string', 'nullable': True,
+                                      'empty': False},
+                    'status': {'type': 'string', 'nullable': True,
+                               'allowed': __import__('flask').current_app.config['INSPIRE_NAME_STATUSES']},
```

Comment:
I'll move to using KB when it's ready.

Thanks for pointing out that there is such thing available